### ISSUE TITLE
TWRP 3.0.2-0 on Angler NPD56N - stuck on splash screen

### ISSUE THREAD

**Issue by dpedu** [TeamWin/Team-Win-Recovery-Project#704]
I updated my 6P to the Developer Preview 4 of android N (NPD56N). I flashed TWRP this morning and when entering recovery it hangs forever on the TWRP splash screen.

**Comment by Flat** [comment]
Same Issue. 3.0.0 Seems to load fine though.

**Comment by dpedu** [comment]
Thanks for the tip - 3.0.0 is working for me too.

**Comment by Johnjr00** [comment]
Same issue here too, but 3.0.0.0 doesn't work for me. 

My 6p is encrypted and TWRP 3.0.0.0 wouldn't decrypt /system. I tried "default_password" as some people suggested online, but that didn't work. Anyone know if there's a different password I can try? I have a pattern and fingerprint set up in Android if that means anything... 

I also read that TWRP 3.0.0-1 fixes the decryption, so I tried that and it got stuck on the splash screen just like on 3.0.2-0.

Would appreciate any suggestions here. Thanks in advance.

**Comment by Grimmity** [comment]
Having the same issue and also on NPD56N.

3.0.0 wasn't working for decryption either. 3.0.2 was working right before I flashed NPD56N

**Comment by dabele91** [comment]
Same problem on NPD90G.
Does anyone have a solution to the problem?

**Comment by Flat** [comment]
Yeah. Don't use TWRP. I've been using FlashFire by Chainfire and it does everything I need without a hacky custom recovery.

**Comment by dabele91** [comment]
The problem is that i want to flash SuperSU and therefore i need a custom recovery.

**Comment by Flat** [comment]
If your phone supports side loading from adb you can sideload the cf-autoroot.zip from adb.

**Comment by dabele91** [comment]
As far as i know, cf-autoroot wipes my user data (Nexus 6P encrypted).
So this isn't a good solution for me.

**Comment by Flat** [comment]
If you've already unlocked your bootloader it shouldn't.

**Comment by dpedu** [comment]
By the way, TWRP 3.0.2-1 (on a nexus 6p running the public release on android N) succeeded for me where prior versions have not worked - flashing supersu onto an encrypted device.
